integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/config/sequences/iterate/iterateLogAndSendSequence

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sequence xmlns="http://ws.apache.org/ns/synapse" name="iterateLogAndSendSequence">
+	<in>
```

Comment:
shall we avoid the usage of in and out mediators

Reply:
done